vue-frontend/src/components/error404/index.vue

```diff
@@ -19,7 +19,7 @@
       </div>
       <router-link class="tw-my-16 gradient-border-btn" to="/">
         <font-awesome-icon
-          class="fa tw-text-[#f2709c] tw-text-[1.5rem] tw-mr-[16px]"
+          class="arrow tw-w-4 fa tw-text-[#f2709c] tw-text-[1.1rem] tw-mr-[5px]"
```

Comment:
On refresh page button text showing in next line 

[Screencast from 20-09-22 03:04:28 PM IST.webm](https://user-images.githubusercontent.com/94360312/191224452-dd44a318-3571-419e-8ccc-f17a3405ef2e.webm)